tests/qgis-projects/tests/form_edition_value_relation_field.qgs.cfg

```diff
@@ -0,0 +1,237 @@
+{
+    "metadata": {
+        "lizmap_plugin_version": 30209,
```

Comment:
@rldhont Your lizmap plugin version is outdated.

Reply:
updated